echoregions/lines/lines.py

```diff
@@ -98,48 +76,54 @@ def swap_val(val: Union[int, float]) -> Union[int, float]:
         if self.nan_depth_value is None:
             return
 
-        regions = self.data if inplace else self.data.copy()
+        regions = self._data if inplace else self._data.copy()
         regions.loc[:] = regions.apply(replace_depth, axis=1)
```

Comment:
I wonder if we can replace the whole `replace_depth`/`swap_val` thing by a lambda function, since what it does is to swap out -10000.99 with `self.nan_depth_value`.

Reply:
Can also define `ECHOVIEW_NAN_DEPTH_VALUE = -10000.99` near the top of the script and use this constant in place of the value, so that it is easier to keep track of numbers like this flying around.

See example here: https://github.com/OSOceanAcoustics/echopype/blob/9b401ca8250e60c7264b274e66145d0f7842def6/echopype/echodata/combine.py#L20-L26